Tests are still failing :(

This is `tensorflow/python:image_ops_test:FlipTransposeRotateTest.testFlipImageUnknownShape`
```
Traceback (most recent call last):
  File ".../tensorflow/python/client/session.py", line 1365, in _do_call
    return fn(*args)
  File ".../tensorflow/python/client/session.py", line 1350, in _run_fn
    target_list, run_metadata)
  File ".../tensorflow/python/client/session.py", line 1443, in _call_tf_sessionrun
    run_metadata)
google3.third_party.tensorflow.python.framework.errors_impl.InvalidArgumentError: TypeError: `generator` yielded an element that could not be converted to the expected type. The expected type was int32, but the yielded element was Tensor("Const:0", shape=(1, 2, 2, 3), dtype=int32).
Traceback (most recent call last):

  File ".../tensorflow/python/data/ops/dataset_ops.py", line 843, in generator_py_func
    ret, dtype=dtype.as_numpy_dtype))

  File ".../tensorflow/python/ops/script_ops.py", line 205, in _convert
    result = np.asarray(value, dtype=dtype, order="C")

  File ".../numpy/core/numeric.py", line 538, in asarray
    return array(a, dtype, copy=False, order=order)

TypeError: __array__() takes 1 positional argument but 2 were given
```

--- a/tensorflow/python/ops/image_ops_test.py
+++ b/tensorflow/python/ops/image_ops_test.py
@@ -31,11 +31,13 @@ from six.moves import xrange  # pylint: disable=redefined-builtin
 from tensorflow.core.protobuf import config_pb2
 from tensorflow.python.client import session
 from tensorflow.python.compat import compat
+from tensorflow.python.data.experimental.ops import get_single_element
 from tensorflow.python.data.ops import dataset_ops
 from tensorflow.python.framework import constant_op
 from tensorflow.python.framework import dtypes
 from tensorflow.python.framework import errors
 from tensorflow.python.framework import ops
+from tensorflow.python.framework import tensor_shape
 from tensorflow.python.framework import test_util
 from tensorflow.python.ops import array_ops
 from tensorflow.python.ops import control_flow_ops
@@ -1375,10 +1377,13 @@ class FlipTransposeRotateTest(test_util.TensorFlowTestCase):
     def generator(): yield image_input
 
     dataset = dataset_ops.Dataset.from_generator(
-        generator, output_types=dtypes.int32)
+        generator,
+        output_types=dtypes.int32,
+        output_shapes=tensor_shape.TensorShape([1, 2, 2, 3]))
     dataset = dataset.map(image_ops.flip_left_right)
 
-    image_flipped_via_dataset_map = next(iter(dataset))
+    image_flipped_via_dataset_map = get_single_element.get_single_element(
+        dataset.take(1))
     self.assertAllEqual(image_flipped_via_dataset_map, expected_output)
 
 class AdjustContrastTest(test_util.TensorFlowTestCase):

